pages/introduction/about/README.md

```diff
@@ -28,13 +28,13 @@ For more information, see [www.access-for-all.ch](http://access-for-all.ch/).
 
 On the other side of the «Accessibility Alliance», there are several acclaimed web agencies. They provided the technical solution and visual design for the guide, gave support in marketing it, etc.
 
-### Hinderling Volkart
+### Dept
 
-![Hinderling Volkart logo](_media/hv.png){.agency-logo}
+![Dept logo](_media/dept.png){.agency-logo}
```

Comment:
@christophdubach Shouldn't that point to `/img/logo/dept.png` like in the footer?

Reply:
I honestly don't know how the markdown parsing of the images work here, but it seems like that image just doesn't exist, so you might have to add to a second folder?

Reply:
Never mind. I don't know how it works that the images are also in a `_media` folder.

Reply:
@christophdubach, @renestalder, could this have been a caching issue? I'm seeing the logo on https://deploy-preview-381--accessibility-developer-guide.netlify.app/introduction/about/

Reply:
@backflip I think he fixed it meanwhile.